proposals/new-project-addition-process.md

```diff
@@ -0,0 +1,36 @@
+## Motivation
+As Kubeflow has become a well established ecosystem and community, several
+projects may want to be incorporated into Kubeflow in different capacities.
+
+That may include:
+1. Project Donation
+2. Inclusion in the Manifest
+3. Inclusion in documentation as an add on
+
+This proposal aims to give guidelines and set expectations for community
+members about the application process.
+
+## Goals
+The goal of this proposal is to provide a clear process for being included in
+the Kubeflow ecosystem.
+
+## Non-Goals
+Give specific recommendations for evaluating any individual project.
+
+## UI or API
+N/A
+
+## Design
+Project owners or maintainers will apply for inclusion by following a four
+step process. submitting the
+
+The process is outlined in 4 steps:
```

Comment:
We might want to also ask to:
- Send announcement in `kubeflow-discuss`
- Ask Kubeflow Outreach Committee to publish message in Slack, LinkedIn, Twitter Kubeflow resources